lib/init.js

```diff
@@ -31,6 +31,19 @@ const findElixirProjectPath = async (editorPath) => {
   const editorDir = dirname(editorPath);
   const mixexsPath = find(editorDir, 'mix.exs');
   if (mixexsPath !== null) {
+    const pathArray = mixexsPath.split('/');
+    if (pathArray.length > 3 && pathArray[pathArray.length - 3] === 'apps') {
+      //  Treat this as an umbrella app. This may be wrong -
+      //  If you happen to keep your code in a directory called 'apps'
+      pathArray.splice((pathArray.length - 3), 3);
+      const umbrellaProjectPath = pathArray.join('/');
```

Comment:
I've amended the second commit there. Thanks for pointing me to the Node documentation - I didn't realise that was were `path.js` was coming from.